src/packages/@ncigdc/containers/CartPage.js

```diff
@@ -127,15 +153,86 @@ const CartPage: TCartPage = ({ viewer, files, user, theme } = {}) => {
                   style: { textAlign: 'right' },
                 },
                 {
+                  key: 'case_count_meter',
+                  title: (
+                    <Link
+                      merge="replace"
+                      pathname="/repository"
+                      query={{
+                        filters: makeFilter(
```

Comment:
I'm not sure this abstraction is helpful :D 
I think writing the actual filter would be clearer
```
{
    op: 'or',
    content: files.map(file => ({
        op: 'in',
        content: {
            field: 'files.file_id',
            value: [file.file_id],
        }
    }))
}
```